### ISSUE TITLE
can tensortRT  run module distribute  like torch.distributed  to use many GPU? 

### ISSUE THREAD

**Issue by yulinhuyang** [NVIDIA/TensorRT#1105]
## Environment

**TensorRT Version**: 
**NVIDIA GPU**: 
**NVIDIA Driver Version**: 
**CUDA Version**: 
**CUDNN Version**: 
**Operating System**: 
**Python Version (if applicable)**: 
**Tensorflow Version (if applicable)**: 
**PyTorch Version (if applicable)**: 
**Baremetal or Container (if so, version)**:

**Comment by ttyio** [comment]
Hello @yulinhuyang , we cannot run single inference on different GPUs, because inference is lightweight than training. However, you can have a request queue in your system, and assign each request to different GPUs by your scheduler.

**Comment by yulinhuyang** [comment]
I get it . i used nn.parallel.parallel_apply  in torch.  maybe it should  implement again in tensorRT. many thanks for you.~

**Comment by ttyio** [comment]
closing since no activity for more than 3 weeks, please reopen if you still have question, thanks!
